src/CodeEditor/services/Monaco/ATA.ts

```diff
@@ -30,16 +30,9 @@ const make = Effect.gen(function* () {
     }
   })
 
-  monaco.languages.typescript.typescriptDefaults.setCompilerOptions({
-    allowNonTsExtensions: true,
-    allowSyntheticDefaultImports: true,
-    moduleResolution: monaco.languages.typescript.ModuleResolutionKind.NodeJs,
-    strict: true,
-    target: monaco.languages.typescript.ScriptTarget.ESNext,
-    strictNullChecks: true
-  })
-
   const install = (editor: monaco.editor.IStandaloneCodeEditor) => {
+    // TODO: auto-load from package json in addition to editor
+    ata(`import * as Schema from "@effect/schema/Schema"`)
```

Comment:
@tim-smart - I'd like to see if we can enhance the editor preload to perform ATA on packages in the `package.json` as well as what's in the current editor's model. This will be required to allow auto-importing to work.